File: src/main/java/com/haozileung/web/service/permission/IMenuService.java
package com.haozileung.web.service.permission;

import com.haozileung.infra.pager.PageResult;
import com.haozileung.web.domain.permission.Menu;

public interface IMenuService {

    void save(Menu dictionary);

    void update(Menu dictionary);

    void delete(Menu dictionary);

    PageResult<Menu> query(Menu dictionary, int pageNo, int pageSize);
}

File: src/main/kotlin/com/haozileung/web/service/permission/IMenuService.kt
package com.haozileung.web.service.permission

import com.haozileung.infra.pager.PageResult
import com.haozileung.web.domain.permission.Menu

interface IMenuService {

    fun save(dictionary: Menu)

    fun update(dictionary: Menu)

    fun delete(dictionary: Menu)

    fun query(dictionary: Menu, pageNo: Int, pageSize: Int): PageResult<Menu>
}
